task-sdk/tests/test_public_api.py

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Maybe in a follow up, move this to a better location, like: `task-sdk/tests/docs`

Reply:
Yes I can do it as part of https://github.com/apache/airflow/pull/51153#discussion_r2139672481